src/material-experimental/mdc-tooltip/tooltip.spec.ts

```diff
@@ -728,6 +728,9 @@ describe('MDC-based MatTooltip', () => {
       tick(500);
 
       expect(overlayContainerElement.querySelector('.mat-mdc-tooltip')).toBeNull();
+
+      // Flush due to the additional tick that is necessary for the FocusMonitor.
+      flush();
```

Comment:
These `flush` calls are because this test only ticks 500 ms, but the `FocusMonitor` creates a timer for 650 ms (`TOUCH_BUFFER_MS`) for touch events. Ticking for 650 ms, or adding this `flush`, allows the pending timer to complete. Previously this timer was only created for `touchstart`, not `focusVia(el, 'touch')`, now it's created for both (to match the behavior of keyboard and mouse).

Reply:
It's worth noting that the last time we did something like this in the `FocusMonitor`, it was difficult to land in g3, because  it broke tests using `fakeAsync`.

Reply:
Ack. I'll see what things look like in g3 and rework things if needed.

Reply:
I tested the change in g3 and things fortunately look good.

Reply:
Note: These `flush` calls ended up being removed to keep `FocusMonitor`'s existing behavior and because ultimately not clearing the origin after 650 ms when using `focusVia` is unnecessary (see https://github.com/angular/components/pull/22489#discussion_r617097268).